ocd-3393.sql

```diff
@@ -0,0 +1,39 @@
+-- This needs to be dropped because the may have been updated to include the old names for
+-- the RWT fields.
+DROP VIEW IF EXISTS openchpl.certified_product_details CASCADE;
+
+ALTER TABLE openchpl.certified_product
+DROP COLUMN IF EXISTS rwt_plan_url;
```

Comment:
These change the tables as they're defined in the data-model, right? So shouldn't this PR have a change to the dev/data-model.sql file as well?